lib/perl/Genome/Model/SomaticValidation/Command/AlignmentStatsSummary.pm

```diff
@@ -144,7 +144,8 @@ sub _alignment_metrics_from_result {
         $total_bases += $lane->total_base_count;
         $total_mapped_bases += $lane->total_aligned_base_count;
         my $filt_error_rate_avg;
-        if ($mismatches ne "NA" and $lane->instrument_data->can("filt_error_rate_avg")) {
+        if ($mismatches ne "NA" and $lane->instrument_data->can("filt_error_rate_avg")
+            and defined($lane->instrument_data->can("filt_error_rate_avg"))) {
```

Comment:
Thanks for the catch - awesome to be introducing bugs and removing the test that would potentially catch them at the same time :-\